src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,24 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMUM_PASSWORD_LENGTH = 6;
+    private static final int MINIMUM_USER_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RuntimeException("User cannot be null!");
+        }
+        if (user.getPassword().length() < MINIMUM_PASSWORD_LENGTH) {
```

Comment:
what if password is null, this would produce NullPointerException
this exception is never to be tested - it should not appear